test/test-ghe-restore.sh

```diff
@@ -466,8 +466,57 @@ begin_test "ghe-restore cluster"
   grep -q "Pages to git-server-fake-uuid2" "$TRASHDIR/restore-out"
   grep -q "dpages-cluster-restore-finalize OK" "$TRASHDIR/restore-out"
 
+  # verify no warnings printed
+  grep -qv "Warning" "$TRASHDIR/restore-out"
```

Comment:
This will always succeed if the `restore-out` file contains more than just "Warning" lines:

```console
$ cat foo.out
Warning
baaa
$ grep -qv Warning foo.out; echo $?
0
$
```

You would need to negate a successful find:

```console
$ grep -q Warning foo.out; echo $?
0
$ ! grep -q Warning foo.out; echo $?
1
$ ! grep -q foo foo.out; echo $?
0
$
```

Reply:
Ah, thanks for that @lildude 🙇 Fix coming up.